data.sql

```diff
@@ -1,6 +1,51 @@
 /* Populate database with sample data. */
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Charmander', '08-02-2020', 0, false, -11);
 
-INSERT INTO animals VALUES (1, 'Agumon', '02-03-2020', 0, true, 10.23);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '15-11-2018', 2, true, 8);
-INSERT INTO animals VALUES (3, 'Pikachu', '07-01-2021', 1, false, 15.04);
-INSERT INTO animals VALUES (4, 'Devimon', '12-05-2017', 5, true, 11);
\ No newline at end of file
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Plantmon', '15-11-2022', 2, true, -5.7);
+
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Squirtle', '2-04-1993', 3, false, -12.13);
+
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Angemon', '12-06-2005', 1, true, -45);
+
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Boarmon', '07-06-2005', 7, true, 20.4);
+
+INSERT INTO animals 
+              (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+            VALUES
+              ('Blossom', '13-10-1998', 3, true, 17);
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species != 'digimon';
+COMMIT;
+
+BEGIN TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '01-01-2022';
+SAVEPOINT savepoint1;
+UPDATE animals SET weight_kg = weight_kg*-1 WHERE weight_kg < 0;
+ROLLBACK TO savepoint1;
+UPDATE animals SET weight_kg = weight_kg*-1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
after `UPDATE` we need to verify change was made and after `ROLLBACK` we also want to verify that change was undone 

- [ ] Kindly update query
- [ ] Please move the transaction queries to the `queries.sql` file